Sources/LinkedIn/API/LinkedInAPI.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LinkedInAPI.swift
+//  PovioKitAuth
+//
+//  Created by Borut Tomazin on 04/09/2023.
+//  Copyright © 2023 Povio Inc. All rights reserved.
+//
+
+import Foundation
+import PovioKitNetworking
+
+public struct LinkedInAPI {
```

Comment:
Any reason this is a struct? I would change it to a class instead.

Reply:
What do we gain with class?

Reply:
When structs contain objects, there is more RC incrementing / decrementing when copying an instance of a struct; whereas with classes, if you assign an object to another, only a RC to the object itself is incremented.

Reply:
I usually prefer classes for all manager/service/api kind of stuff. Not really sure how this sneaked in. I'll update. Thanks for the notice 👌